server/src/main/java/org/apache/iotdb/db/engine/StorageEngine.java

```diff
@@ -609,6 +609,15 @@ public boolean isFileAlreadyExist(TsFileResource tsFileResource, String storageG
     return processor != null && processor.isFileAlreadyExist(tsFileResource, partitionNum);
   }
 
+  public int countTsFileProcessors() {
```

Comment:
This count may be not very accurate. If you want accurate count, consider synchronize. If you need performance and accept inaccurate count, this way is better.

Reply:
This method is no longer useful. I'll remove it in the next commit. :)